common/components/controllers/AboutProjectController.jsx

```diff
@@ -212,14 +212,14 @@ class AboutProjectController extends React.PureComponent<{||}, State> {
               <Grid className='AboutProjects-description-details'>
                 <div id='project-details'>{project.project_description}</div>
               <Grid className='AboutProjects-skills-container' container direction='row'>
-                <div className='AboutProjects-skills'>
+                {project && !_.isEmpty(project && project.project_positions) && <div className='AboutProjects-skills'>
```

Comment:
For multi-line conditionally rendered blocks, we should break that out into their own render functions for readability's sake.